xml/planning-core_non-core_openstack_support.xml

```diff
@@ -47,81 +47,81 @@
        <entry>Yes</entry>
        <entry>Yes</entry>
        <entry>  </entry>
-       <entry>Freezer</entry>
+       <entry>&o_img;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
       </row>
       <row>
-       <entry>&o_img;</entry>
+       <entry>&o_orch;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
        <entry>  </entry>
-       <entry>&o_orch;</entry>
+       <entry>&o_dash;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
       </row>
       <row>
-       <entry>&o_dash;</entry>
+       <entry>&o_iron;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
        <entry>  </entry>
-       <entry>&o_iron;</entry>
+       <entry>&o_ident;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
       </row>
       <row>
-       <entry>&o_ident;</entry>
+       <entry>&o_container;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
        <entry>  </entry>
-       <entry>&o_container;</entry>
+       <entry>&o_sharefs;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
       </row>
       <row>
-       <entry>&o_sharefs;</entry>
+       <entry>&o_monitor;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
        <entry>  </entry>
-      <entry>&o_monitor;</entry>
+      <entry>&o_monitor;-&o_meter;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
       </row>
       <row>
-       <entry>&o_monitor;-&o_meter;</entry>
+       <entry>&o_netw;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
        <entry>  </entry>
-       <entry>&o_netw;</entry>
+       <entry>&o_netw;(LBaaSv2)</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
       </row>
       <row>
-       <entry>&o_netw;(LBaaSv2)</entry>
+       <entry>&o_netw;(VPNaaS)</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
        <entry>  </entry>
-       <entry>&o_netw;(VPNaaS)</entry>
+       <entry>&o_netw;(FWaaS)</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
       </row>
       <row>
-       <entry>&o_netw;(FWaaS)</entry>
+       <entry>&o_comp;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
        <entry>  </entry>
-       <entry>&o_comp;</entry>
+       <entry>Octavia</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
       </row>
       <row>
-       <entry>Octavia</entry>
-       <entry>Yes</entry>
-       <entry>Yes</entry>
-       <entry>  </entry>
        <entry>&swift;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
+       <entry>  </entry>
```

Comment:
Out of curiosity: What are these  empty `<entry></entry>`?

Reply:
I imagine they generate the remainder of the table? Correct @csymons-suse ?

Reply:
Oh potentially not, it doesn't generate properly

Reply:
It builds properly with DAPS. In the rendered table, it inserts a small blank column that makes the table readable. This was a conversion from an ugly artifact in the original HOS documentation. I tried it with and without the blank column. Preferred this look.

Might be a moot point as there are now "conflicts that must be resolved". This pull request is almost a month old. Ungood.

Reply:
I see. Thanks for the explanation!